### ISSUE TITLE
A11Y Lab - Info and Relationships

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#157]
Info and Relationships: Information, structure, and relationships conveyed
through presentation can be programmatically determined or are available
in text.
https://
ecp273mv5gooagvmealdr7ypnikpfso3epgtffgxf43d2xtiahdjrxad.onion.try.hushline.app/
The input or textarea must have a visible label related.
Visible labels benefit all users. When form fields do not have labels that are meaningful
and visible to all users, at all times, users might have to guess or infer what goes in
them.
Placeholder text must not be used as the only method of providing a label for a text
input, placeholder it is not enough because it is not compatible with all screen
readers and does not have enough contrast.

**Comment by glenn-sorrentino** [comment]
👍 fixed - added a label for textarea

**Comment by glenn-sorrentino** [comment]
https://github.com/scidsg/project-info/blob/main/hush-line/1.%20Accessibility/WCAG%202.1%20AAA_HushLine%20website.pdf
